etc/helm/pachyderm/templates/pachd/deployment-id-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- /*
+SPDX-FileCopyrightText: Pachyderm, Inc. <info@pachyderm.com>
+SPDX-License-Identifier: Apache-2.0
+*/ -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: deployment-id-secret
+    suite: pachyderm
+  name: deployment-id-secret
```

Comment:
I'd prefix this with pachyderm to make it clear who the secret belongs to when getting secrets from the namespace